stdlib/public/RemoteInspection/TypeLowering.cpp

```diff
@@ -437,14 +437,26 @@ BitMask RecordTypeInfo::getSpareBits(TypeConverter &TC, bool &hasAddrOnly) const
     // Mask the rest of the fields as usual...
     break;
   }
-  case RecordKind::ClassExistential:
-    // Class existential is a data pointer that does expose spare bits
-    // ... so we can fall through ...
+  case RecordKind::ClassExistential: {
+    // First pointer in a Class Existential is the class pointer
+    // itself, which can be tagged or have other mysteries on 64-bit, so
+    // it exposes no spare bits from the first word there...
+    auto pointerBytes = TC.targetPointerSize();
+    if (pointerBytes == 8) {
```

Comment:
If anyone is trying to get RemoteMirror to work on non-Darwin, they may need to tweak this some more.